dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -649,6 +658,20 @@ Resources:
       VpcId: !Ref VPC
       InternetGatewayId: !Ref VPCInternetGateway
 
+  VPCNatGatewayElasticIP:
+    Condition: DeployAGwithVPC
+    Type: AWS::EC2::EIP
+    Properties:
+      Domain: VPC
+
+  VPCNatGateway:
+    Condition: DeployAGwithVPC
+    Type: "AWS::EC2::NatGateway"
+    DependsOn: VPCNatGatewayElasticIP
+    Properties:
+      AllocationId: !GetAtt VPCNatGatewayElasticIP.AllocationId
+      SubnetId: !Ref VPCPublicSubnet1
+
```

Comment:
Mostly a note to me, please check if I read this right:
*AG EC2 (:621) & Lambda(:83) now run in the private subnet of the VPC.
*private(:635) and public(:644) Subnets are in same VPC
*NATGateway is in Public subnet (673), with an elastic IP (672)
*all traffic(708) from private subnet goes through NATGateway(709) in public (via routing table)

I'm wondering how is NATGateway from Public subnet visible in the Private one, but I assume that there is some logic/magic that happens behind the line 709 that NatGatewayId, since AWS has a dedicated field for that.